src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,688 @@
+              <p>- Countdown on final presentation </p>
+            </aside>
+          </section>
+        </section>
+
+        <!--SLIDE 11-->
+        <!--Assignment time-->
+        <section>
+          <!--SLIDE 11.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>Let's fill the seats!</h3>
+
+            <aside class="notes">
+              Send an assigmnet link.
+            </aside>
+          </section>
+
+          <!--SLIDE 11.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>Let's fill the seats!</h3>
+            <p class="small-please" style="font-style: italic;">Teacher will send you the link and password through our Zoom chat.</p>
+
+            <aside class="notes">
+              Send an assigmnet link.
+            </aside>
+          </section>
+        </section>
+
+        <!--SLIDE 12-->
+        <!--Introduction to CSS (bullet points)-->
+        <section>
+          <!--SLIDE 12.1-->
+          <section data-auto-animate data-timing="10">
+            <h2>Introduction to CSS</h2>
+          </section>
+
+          <!--SLIDE 12.2-->
+          <section data-auto-animate data-timing="10">
+            <h2>Introduction to CSS</h2>
+            <p>What's CSS?</p>
+          </section>
+
+          <!--SLIDE 12.3-->
+          <section data-auto-animate data-timing="300">
+            <h3>Quick Introduction to CSS</h3>
+            <p>What's CSS?</p>
+
+            <ul>
+              <li>CSS is the code that styles your web content</li>
+              <li>CSS stands for <b>C</b>ascading <b>S</b>tyle <b>S</b>heets</li>
+              <li>Why knowing <b>CSS</b></li>
+              <li>
+                <a href="https://www.w3.org/standards/webdesign/htmlcss.html" target="_blank">W3C</a> standards
+              </li>
+            </ul>
+          </section>
+        </section>
+
+        <!--SLIDE 13-->
+        <!--Inline CSS (examples)-->
+        <section>
+          <!--SLIDE 13.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>For now, let's use "inline" CSS</h3>
+          </section>
+
+          <!--SLIDE 13.2-->
+          <section data-auto-animate data-timing="10">
+            <h3>For now, let's use "inline" CSS</h3>
+            <img data-src="images/wat.jpg" class="r-frame" />
+          </section>
+
+          <!--SLIDE 13.3-->
+          <section data-auto-animate data-timing="300">
+            <h3>For now, let's use "inline" CSS</h3>
+            <p class="small-please">You most likely remember our tag attributes from some of the tags, don't ya?</p>
+
+            <p class="fragment small-please" style="color: #c98f23; margin-bottom: 0;">Still, why "inline" and what does that mean❓</p>
+            <p class="fragment small-please current-value"  style="color: #0d407a; margin-top: 0;">– It means we want to make sure we add styles for a few components to get an idea of how CSS works</p>
```

Comment:
This little hints are a bit small and the color of the second one is kinda dark taking in account the dark bg. 
I would add the definition of inline as a regular text instead of a hint and in the next slide the other part as suggested by sebastian :)

Reply:
I have fixed them as well.